client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -359,8 +359,9 @@ class CancelPurchaseForm extends React.Component {
 			? this.state.questionOneText
 			: 'select_a_product';
 
-		const appendRadioOption = ( key, radioPrompt, textPlaceholder ) =>
+		const appendRadioOption = ( groupName, key, radioPrompt, textPlaceholder ) =>
```

Comment:
Probably out of scope, but why do you think about extracting the duplicate `appendRadioOption` function? We define it once for each question with very small changes...